config/routechangerconfig.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+type routeChangerConfig struct {
+	Reg  *regexp.Regexp
+	Repl string
+}
+
+func (rcc routeChangerConfig) String() string {
+	if rcc.Reg == nil {
```

Comment:
this if might not be necessary here. We can just return whatever the user wrote.

Reply:
it's workaround for getting <NIL> string in some outputs